bazel/bazel_test.go

```diff
@@ -91,3 +91,21 @@ func TestCancel(t *testing.T) {
 	b := New()
 	b.Cancel()
 }
+
+func TestBazelNpmPath(t *testing.T) {
+	// Where ibazel gets installed by npm
+	ibazelPath := os.Getenv("TEST_TMPDIR") + "/node_modules/@bazel/ibazel/bin/linux-amd64/ibazel"
```

Comment:
Can you turn this into a [table driven test](https://github.com/golang/go/wiki/TableDrivenTests), and add a few negative test cases?